cves/kernel/CVE-2017-6074.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The dccp_rcv_state_process function in /net/dccp/input.c which listens for inputs 
+  freed the socket buffer when it received a DCCP_PKT_REQUEST type request for a 
+  packet. This enabled local users to gain unprivalaged access to the kernal's 
+  memory.
```

Comment:
Same as above - make it more accessible to SE sophmores who don't know this code. explain sockets, what this exchange is about, and what it means to get access to kernel memory. This is a complex one - research all the keywords here.

Also, it's worth mentioning here the stuff you mentioned about the fuzzer later on.

Reply:
Yep, all good suggestions. Thank you again for the advice!